docs/documentation/maintenance-coordinator.md

```diff
@@ -0,0 +1,91 @@
+#Coordinator Model
```

Comment:
@Amstutz github doesn't render this as headline without a space (see also other headlines below)

Reply:
Thx, fixed.

Reply:
Thx again ;-)